ST_mortality.c

```diff
@@ -353,18 +421,7 @@ void proportion_Recovery(void) {
 
         g = RGroup[rg];
 
-        if ((Globals.currYear >= g->killfreq_startyr) && (g->killfreq > 0.)) {
-            if (g->killfreq < 1.0) {
-                if (RandUni(&mortality_rng) <= g->killfreq) {
-                    g->killyr = Globals.currYear;
-                }
-
-            } else if (((Globals.currYear - g->killfreq_startyr) % (IntU) g->killfreq) == 0) {
-                g->killyr = Globals.currYear;
-            }
-
-        }
-
```

Comment:
@chaukap I assume this has all been deleted as it is redundant? I.e. we have already determined if it is a kill year in mort_EndofYear?

It is really important that biomass is only recovered in the years that are actually fire years.

Reply:
@kpalmqui Yes this code only affects RGroup[rg]->killyr, which has already been set in mort_EndofYear. The recovery functionality should still work the same.